scripts/sync_gokb_packages.groovy

```diff
@@ -71,9 +71,9 @@ while ( moredata ) {
             else if (type == 'BookInstance'|| type == 'Monograph') {
                 newtipp.title.type = 'Book'
             }
-            newtipp.platform = directAddFields (xmltipp.platform, ['primaryUrl'], addCoreItems ( xmltipp.platform ))
             resourceFieldMap['tipps'].add(newtipp)
```

Comment:
This line adds the finished TIPP to the result and needs to be at the very end.

Reply:
Done.